src/libs/blueprint/conduit_blueprint_mesh.hpp

```diff
@@ -742,6 +742,13 @@ namespace matset
     //-------------------------------------------------------------------------
     bool CONDUIT_BLUEPRINT_API is_material_dominant(const conduit::Node &matset);
 
+    //-------------------------------------------------------------------------
+    void CONDUIT_BLUEPRINT_API convert_matset(const conduit::Node &src_matset,
+                                              conduit::Node &dest_matset,
+                                              const std::string &src_matset_type,
```

Comment:
can we detect instead of passing (folks could make a mistake)?

how about method sig variants in the public interface that look like
```
to_full
to_sparse_by_element
to_sparse_by_material
```

Reply:
Ok I have addressed this @cyrush

Reply:
what do you think now @cyrush